### DISCUSSION TITLE
Reset admin

### DISCUSSION THREAD

**Discussion by tjym-2305** [revenz/Fenrus#168] [Q&A]
Hi,
first, thanks for the hard work, i tried a lot of dashboard, but the simplicity of this one get me, so again thanks :)

my question is, due to a humain error (i didn't disabled the registration), and rather than click on login, i clicked on register again with the same admin login to overwrite the admin with a simple user rights, so i want to know if there is anyway back to avoid doing everything from the ground?
thanks for your help.

I installed Fenrus on Qnap NAS equivalent to docker.

**Comment by revenz** [comment]
What version are you on?
